src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -19,9 +24,11 @@
 public class PersonCard extends UiPart<Region> {
 
     private static final String FXML = "PersonListCard.fxml";
-    private static String[] randomColors = { "navy", "teal", "violet", "green", "purple", "black" };
-    private static HashMap<String, String> tagColors = new HashMap<String, String>();
+    private static HashMap<ReadOnlyPerson, String> personColors = new HashMap<>();
+    private static HashMap<String, String> tagColors = new HashMap<>();
     private static Random random = new Random();
+    private static final String defaultThemeTagColor = "#fc4465";
+    private static final double goldenRatio = 0.618033988749895;
```

Comment:
Yes to golden ratio 👍 
Can we have the constant be named in CAPITAL_WITH_UNDERSCORES instead?

Reply:
Updated! 👍✔